spec/influxdb/cases/query_cluster_spec.rb

```diff
@@ -17,57 +17,49 @@
 
   let(:args) { {} }
 
-  describe "POST #create_cluster_admin" do
-    it "creates a new cluster admin" do
-      stub_request(:post, "http://influxdb.test:9999/cluster_admins").with(
-        query: { u: "username", p: "password" },
-        body: { name: "adminadmin", password: "passpass" }
+  describe "#create_cluster_admin" do
+    let(:user) { 'adminadmin' }
+    let(:pass) { 'passpass' }
+    let(:query) { "CREATE USER #{user} WITH PASSWORD '#{pass}' WITH ALL PRIVILEGES" }
+
+    before do
+      stub_request(:get, "http://influxdb.test:9999/query").with(
+        query: {u: "username", p: "password", q: query}
       )
+    end
 
-      expect(subject.create_cluster_admin("adminadmin", "passpass")).to be_a(Net::HTTPOK)
+    it "should GET to create a new cluster admin" do
+      expect(subject.create_cluster_admin(user, pass)).to be_a(Net::HTTPOK)
     end
   end
 
-  describe "POST #update_cluster_admin" do
-    it "updates a cluster admin" do
-      stub_request(:post, "http://influxdb.test:9999/cluster_admins/adminadmin").with(
-        query: { u: "username", p: "password" },
-        body: { password: "passpass" }
-      )
+  describe "#list_cluster_admins" do
+    let(:response) { {"results"=>[{"series"=>[{"columns"=>["user", "admin"], "values"=>[["dbadmin", true], ["foobar", false]]}]}]} }
+    let(:expected_result) { [{"username"=>"dbadmin"}] }
```

Comment:
Well, you might be right. I tried to keep it compatible with old API at first but it changed so much anyway that returning an array here can be a good idea.